src/chart/generateCategoricalChart.tsx

```diff
@@ -2058,6 +2072,19 @@ export const generateCategoricalChart = ({
         return [graphicalItem, null, null];
       }
 
+      if (typeof this.props.defaultActiveIndex !== 'undefined') {
```

Comment:
can't we just check for `this.props.defaultActiveIndex !== undefined` instead of checking `typeof`?